terminus/builders/procedural_city_builder.py

```diff
@@ -16,50 +16,64 @@
 
 import pickle
 
+import os
+import sys
+import subprocess
+
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
 
 class ProceduralCityBuilder(object):
-    def __init__(self, verticesFilename, polygonsFilename):
+    def __init__(self, verticesFilename=None, polygonsFilename=None, size=1500):
         self.verticesFilename = verticesFilename
         self.polygonsFilename = polygonsFilename
+        self.ratio = 100  # ratio between pcg and gazebo (1unit = 100m)
+        self.size = int(size)
```

Comment:
To keep things consistent, can we do this sanitization on the script that interacts with the command line? I would like to keep clean boundaries between the command like script and the model